package.json

```diff
@@ -52,12 +52,12 @@
     "build": "scripts/run-script --parallel build",
     "clean": "scripts/run-script --parallel clean",
     "dev": "scripts/run-script --parallel dev",
-    "dev-test": "jest --bail --watch",
+    "dev-test": "jest --bail --watch \"^(?!.*\\.integ\\.spec\\.js$)\"",
     "posttest": "scripts/run-script test",
     "precommit": "scripts/lint-staged",
     "prepare": "scripts/run-script prepare",
-    "pretest": "eslint --ignore-path .gitignore .",
-    "test": "jest"
+    "test": "jest \"^(?!.*\\.integ\\.spec\\.js$)\"",
+    "test-integration": "jest \".integ\\.spec\\.js$\""
```

Comment:
The Jest config looks a bit complicated for me, maybe we should:
- use different suffixes for unit and integration tests
- create an alternative config file and use it explicitly to run integration tests.

Reply:
I spent some time trying to create a separate configuration file and I couldn't share any config from package.json with the separate config file, and it became a nightmare with babel and the jest config.

Reply:
Ok, I'll take care of this later then.